### DISCUSSION TITLE
How to change the background color of a node in edit text mode

### DISCUSSION THREAD

**Discussion by biswajit-goswami** [freeplane/freeplane#1569] [Help] [answered]
I am using dark theme and whenever I am editing a node, the background color switches to pink ...I dont think I have set such a color anywhere....any hints to change it to some other color?
![image](https://github.com/freeplane/freeplane/assets/11291623/432875d4-e041-4417-9a68-42c0f224bf20)

**Comment by i-plasm** [comment, marked as answer]
Is your node background color transparent/semi-transparent? I've noticed the node in-line core editor will have as background that same color without any transparency (i.e fully opaque). So, if the base color is pink it will show fully opaque. To avoid that, one needs to assign a (mostly) opaque color to the background. ( The background of particular node styles can be modified at : ` Format -> Manage styles -> Edit Styles ` )
![nodecolor](https://github.com/freeplane/freeplane/assets/144830522/20553417-91f9-4971-85bc-bc3846e4964c)
![bg2](https://github.com/freeplane/freeplane/assets/144830522/29ed621f-fe39-4c7d-9482-c49fe1e7d5b3)

**Reply by biswajit-goswami** [reply]
It helped..thanks a lot
